src/Backend/Modules/MediaGalleries/Domain/MediaGallery/MediaGalleryDataGrid.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+namespace Backend\Modules\MediaGalleries\Domain\MediaGallery;
+
+use Backend\Core\Engine\Authentication;
+use Backend\Core\Engine\DataGridDB;
+use Backend\Core\Engine\DataGridFunctions;
+use Backend\Core\Engine\Model;
+use Backend\Core\Language\Language;
+
+/**
+ * @TODO replace with a doctrine implementation of the data grid
+ */
+class MediaGalleryDataGrid extends DataGridDB
+{
+    /**
+     * MediaGalleryDataGrid constructor.
+     */
+    public function __construct()
+    {
+        parent::__construct(
+            'SELECT i.id, i.title, i.action, UNIX_TIMESTAMP(i.publishOn) AS publishOn, UNIX_TIMESTAMP(i.editedOn) AS editedOn
+             FROM MediaGallery AS i'
+        );
+
+        // Setting header labels
+        $this->setHeaderLabels(array(
+            'title' => ucfirst(Language::lbl('Title'))
+        ));
+
+        // Sorting columns
+        $this->setSortingColumns(
+            array(
```

Comment:
short array syntax, also in the rest of the module, best to use php-cs-fixer for that